test/integration/targets/rabbitmq_plugin/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- include: tests.yml
```

Comment:
I would replace this as 
~~~diff
-- include: tests.yml
+- import_tasks: tests.yml
~~~

Reply:
@resmo :+1: